login-fire-common-behavior.html

```diff
@@ -390,6 +406,7 @@
             credential: credential
           });
         }
+        return response;
```

Comment:
The documentation of the function has to be updated according to this change.

Reply:
Oups, forgot to remove it, not needed anymore